packages/ajax/src/interceptors/createXsrfRequestInterceptor.js

```diff
@@ -33,7 +18,7 @@ export async function acceptLanguageRequestInterceptor(request) {
  * @param {Document | { cookie: string }} _document overwriteable for testing
  * @returns {RequestInterceptor}
  */
-export function createXSRFRequestInterceptor(cookieName, headerName, _document = document) {
+export function createXsrfRequestInterceptor(cookieName, headerName, _document = document) {
```

Comment:
Nitpick: I tend to prefer to keep abbreviations all-capsed for better readability, but this is acceptable for me too.

Reply:
Changing `createXSRFRequestInterceptor` to `createXsrfRequestInterceptor`, followed the camel case convention we use,  like ING -> `getIngSth`, I can revert it if this is not consistent with the rest of the code.